roles/edpm_bootstrap/tasks/fips.yml

```diff
@@ -61,7 +61,7 @@
             mode: "0600"
         - name: Call edpm_reboot role
           vars:
-            edpm_reboot_force_reboot: true
+            edpm_reboot_strategy: force
```

Comment:
are we sure that a forced rebbot is something well should
enable by default in any of our roles?

Reply:
It was 'forced'  before I changed it to edpm_reboot_force_reboot and now to  edpm_reboot_strategy: force . If this is wrong, this is existing bug in tripleo too.